### PR TITLE
#56: sigmoid를 yolo output에 씌우자

### PR DESCRIPTION
**Pull request by insurgent92**

### CODE REVIEW

File: yolov1.py [hunk 1]
```diff
@@ -182,6 +182,8 @@ def forward(self, x):
         out = self.fc1(out)
         out = self.fc2(out)
         out = out.reshape((-1, 7, 7, ((5) + self.num_classes)))
+        output[:, :, :, 0] = torch.sigmoid(output[:, :, :, 0])  # sigmoid to objness1_output
+        output[:, :, :, 5:] = torch.sigmoid(output[:, :, :, 5:])  # sigmoid to class_output
```
Comment (ssaru) [inline yolov1.py]: 네 확인했습니다.

File: yolov1.py [hunk 2]
```diff
@@ -25,7 +25,7 @@ def __init__(self, params):
         # LAYER 1
         self.layer1 = nn.Sequential(
             nn.Conv2d(3, 64, kernel_size=7, stride=2, padding=3),
-            nn.BatchNorm2d(64), #, momentum=0.01
+            nn.BatchNorm2d(64),  # momentum=0.01
```
Comment (ssaru) [inline yolov1.py]: 해당 주석은 BatchNorm 옵션인거 같습니다.
해당 주석 지워주시면 될 것 같습니다.

File: yolov1.py [hunk 3]
```diff
@@ -135,19 +135,19 @@ def __init__(self, params):
...
         self.fc2 = nn.Sequential(
-            nn.Linear(4096, 7*7*((5)+self.num_classes))
+            nn.Linear(4096, 7 * 7 * ((5) + self.num_classes))
         )
 
         for m in self.modules():
 
             if isinstance(m, nn.Conv2d):
-                #nn.init.xavier_normal_(m.weight)
+                # nn.init.xavier_normal_(m.weight)
```
Comment (ssaru) [inline yolov1.py]: 해당 주석도 kaiming 초기화와 xavier 초기화 차이를 보려고 넣었던 거라
불필요한 주석같습니다.

**Comment by insurgent92** [comment]
Test.py의 sigmoid function도 제거하였습니다.
